api/db/utils.js

```diff
@@ -25,15 +25,15 @@ const getRegion = (random, country) =>
     ? country.regions[Math.floor(Math.random() * country.regions.length)]
     : { shortCode: 'M' };
 
-const getRandomInterests = (interests) =>
-  interests
+const getRandomSubset = (list, maxSize) =>
```

Comment:
This is nice! While we are at it, let's also add a `minSize` here (it can be 0 by default), so we can fix the issue mentioned on the last sentence of [this comment](https://github.com/Tisn/tisn.app/issues/279#issuecomment-705363801).